test/images/agnhost/dns/dns_windows.go

```diff
@@ -17,40 +17,131 @@ limitations under the License.
 package dns
 
 import (
-	"bytes"
-	"os/exec"
+	"fmt"
```

Comment:
line unrelated

Is there any particular reason we don't have `// +build windows` as line 1 of this file?

Reply:
Sure, it felt redundant, we were already relying on the file name suffix for that (conditional compilation can also be done through filename suffix). Added that line now.